src/components/Top/HeaderTop.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import TripHous from '../../img/icons/logo_vector.png';
+import nightIcon from '../../img/icons/Night.png';
+import account from '../../img/icons/Vector.png';
+
+
+function HeaderTop() {  
+    return (
+        <>
+        <header>
+            <div className="top-icon-logo">
+                <img src={TripHous} alt="TripHous" className="icon-logo"/> 
+            </div>
+            <nav className="stays animated-header-line"><a href="#" className="header-link">Stays</a></nav>
+            <nav className="attractions animated-header-line"><a href="#" className="header-link">Attractions</a></nav>
+            <div className="top-icon-night" style={{ backgroundImage: `url('${nightIcon}')` }} ></div>
+            <div className="top-icon-account" style={{ backgroundImage: `url('${account}')` }} ></div>
+            <nav className="top-icon-burger"></nav>
```

Comment:
see how the nav is used. Nav is a navigation menu that wraps the list
https://developer.mozilla.org/ru/docs/Web/HTML/Element/nav
also convert the header elements to the same size
![image](https://user-images.githubusercontent.com/20189313/148702989-ba7bd99c-305b-433f-9073-060ad10b7846.png)